django/publicmapping/redistricting/models.py

```diff
@@ -557,6 +557,15 @@ def renest(self, parent, subject=None, spatial=True):
 
         return True
 
+    def calc_extent(self):
+        """
+        Returns the extent (array of four floats) of the geounits belonging to this geolevel
+        """
+        extent = self.geounit_set.aggregate(Extent('simple'))['simple__extent']
+
+        # Convert the 4-tuple returned by the query to a list
+        return [extent[0], extent[1], extent[2], extent[3]]
```

Comment:
Could also do `return list(extent)`

Reply:
Wow, I totally didn't realize that. Thanks, that's much better.